dwave/system/testing.py

```diff
@@ -19,13 +19,9 @@
 import numpy as np
 import dimod
 import dwave.cloud.computation
-from dwave.system import qpu_graph
 
-try:
-    #Simplest and efficient choice returning low energy states:
-    from greedy import SteepestDescentSampler as SubstituteSampler
-except ImportError:
-    from dimod import SimulatedAnnealingSampler as SubstituteSampler
+from dwave.samplers import SteepestDescentSampler as SubstituteSampler
```

Comment:
If you're doing just a straight up greedy import, you should update some of the code in `MockDWaveSampler` that has a switching behaviour between dimod/greedy. Also test, IIRC.

Reply:
There is actually a lot of cruft to remove associated with legacy versions. I can try to get it all in this version, but IMO might be better as a followup.

Reply:
I ended up just removing it, I keep forgetting how small a package dwave-system is. Didn't take that long to find them all (I think).